src/gobexport/exporter/utils.py

```diff
@@ -192,3 +192,15 @@ def _get_value_from_list(entity, key, default):
         entity = '|'.join([str(d[key]) if d[key] else "" for d in entity if key in d])
 
     return entity
+
+
+def convert_format(input_format, mapping):
+    """
+    Converts one format to another one using mapping.
+    """
+    output_format = {}
+
+    for key, value in mapping.items():
+        output_format[key] = input_format[value]
+
+    return output_format
```

Comment:
return { key: input_format[value] for key, value in mapping.items()}

Reply:
done